website/docs/getting-started/tutorial.md

```diff
@@ -4,12 +4,6 @@ sidebar_position: 4
 
 # Tutorial
 
-:::caution Outdated
-
-Please note that this tutorial is partly outdated and needs to be overhauled.
-
-:::
-
 This tutorial gives a brief introduction to how the tools work together at the example of the [mime-types](https://www.npmjs.com/package/mime-types) NPM package.
```

Comment:
I believe the "outdated" remark also refers to tools missing, [like the advisor](https://github.com/oss-review-toolkit/ort/issues/4219), and not only existing but outdated information. So we should not remove that remark unless also these points are addressed 😉